EarlGrey/Common/GREYAppleInternals.h

```diff
@@ -146,6 +146,17 @@ IOHIDEventRef IOHIDEventCreateDigitizerFingerEvent(CFAllocatorRef allocator,
  *          UITouch objects, and the relevant touch interaction state.
  */
 - (UITouchesEvent *)_touchesEvent;
+
+/**
+ *  Sends a motion began event for the specified subtype.
+ */
+- (void) _sendMotionBegan:(UIEventSubtype)subtype;
```

Comment:
QQ - any OS availability info here?

Reply:
It's a very old API. It has been available since at least iOS 4.0.